packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -358,6 +375,13 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
               />
             </section>
           )}
+          <section>
+            <VisitAttributeTypeFields
+              setVisitAttributes={setVisitAttributes}
+              isMissingRequiredAttributes={isMissingRequiredAttributes}
+              visitAttributes={visitAttributes}
+            />
+          </section>
```

Comment:
And then this bit would be displayed only if the config property is truthy:

```suggestion
 {config.showVisitAttributeTypeFields ? (
    <section>
      <VisitAttributeTypeFields
        setVisitAttributes={setVisitAttributes}
        isMissingRequiredAttributes={isMissingRequiredAttributes}
        visitAttributes={visitAttributes}
      />
    </section>
  ) : null}
```

Reply:
Yes, exactly @denniskigen !